src/python/pants/backend/python/rules/pex.py

```diff
@@ -248,12 +261,22 @@ def log(self, *args, **kwargs) -> None:
     else:
         argv.append("--no-manylinux")
 
+    argv.append("--no-compile")
```

Comment:
This shouldn't be necessary. Reproducible PEXes are on by default in Pex 2.x :)

Reply:
Ok, great!